amida-tech/amida-messaging-microservice
# Feature/add UUID v2

If this PR fixes a bug, you _must_ add test cases representative of the bug.
#### What's this PR do?

1. Adds migrations for table creation (already in release/2.3.0)
2. update package.json for jenkins unit tests, use .env.test
3. add user endpoint to create user in user table, using jwtToken and payload(for orange, this is done on account creation now)
4. update delete db to delete only if it exists
5. remove service account creation from migration, use admin service account from push notifications instead for security purposes

#### Related JIRA tickets:
https://jira.amida.com/browse/DEVOPS-308
https://jira.amida.com/browse/ORANGE-910
https://jira.amida.com/browse/ORANGE-911

#### How should this be manually tested?

1. start auth microservice  and orange microservice
2. `yarn`
3. `yarn migrate` 
4. `yarn start`
5. create new users, verify the username and uuid is in user table (verify uuid matches auth)
